ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -139,6 +149,14 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate,
 
     final TableDestination tableDestination = new TableDestination(tableSpec, null,
         TIME_PARTITIONING);
+
+    if (tableRowFormat != null && tableRowFormat.get() == TableRowFormat.raw) {
+      TableRow tableRow = new TableRow();
+      AddMetadata.attributesToMetadataPayload(message.getAttributeMap()).forEach(tableRow::set);
```

Comment:
`AddMetadata.attributesToMetadataPayload` doesn't currently include namespace, doctype, or version, which will be probably the most interesting information for the landfill_raw cases. We may want to add those to `URI_ATTRIBUTES` so that they appear under `metadata.uri.document_type`, etc.

Reply:
it includes those in `IDENTIFYING_FIELDS`, which are put directly in `metadata`. I've added expanded the test to demonstrate this.

that said, `AddMetadata.attributesToMetadataPayload` won't work for landfill, so I've made a distinction between `raw`, `decoded`, and `payload` formats.

Reply:
Very good. I had totally forgotten about `IDENTIFYING_FIELDS`.